pkg/cloud/azure/actuators/machine/actuator.go

```diff
@@ -38,35 +39,38 @@ import (
 type Actuator struct {
 	*deployer.Deployer
 
-	client client.ClusterV1alpha1Interface
+	client     client.ClusterV1alpha1Interface
+	coreClient controllerclient.Client
 }
 
 // ActuatorParams holds parameter information for Actuator.
 type ActuatorParams struct {
-	Client client.ClusterV1alpha1Interface
+	Client     client.ClusterV1alpha1Interface
+	CoreClient controllerclient.Client
 }
 
 // NewActuator returns an actuator.
 func NewActuator(params ActuatorParams) *Actuator {
 	return &Actuator{
-		Deployer: deployer.New(deployer.Params{ScopeGetter: actuators.DefaultScopeGetter}),
-		client:   params.Client,
+		Deployer:   deployer.New(deployer.Params{ScopeGetter: actuators.DefaultScopeGetter}),
+		client:     params.Client,
+		coreClient: params.CoreClient,
 	}
 }
 
 // Create creates a machine and is invoked by the machine controller.
 func (a *Actuator) Create(ctx context.Context, cluster *clusterv1.Cluster, machine *clusterv1.Machine) error {
-	klog.Infof("Creating machine %v for cluster %v", machine.Name, cluster.Name)
+	klog.Infof("Creating machine %v", machine.Name)
 
-	scope, err := actuators.NewMachineScope(actuators.MachineScopeParams{Machine: machine, Cluster: cluster, Client: a.client})
+	scope, err := actuators.NewMachineScope(actuators.MachineScopeParams{Machine: machine, Cluster: cluster, Client: a.client, CoreClient: a.coreClient})
```

Comment:
This is a bit of a nit, but this line is really long. I think it would be more readable for the `MachineScopeParams` literal to be multi-line.